lib/sgtn-client/api/translation.rb

```diff
@@ -38,7 +44,10 @@ def self.getString_p(component, key, plural_args, locale)
         end
       end
 
-      def self.getString_f(component, key, args, locale, *optionals)
+      def self.getString_f(component, key, args, locale=SgtnClient::Config.configurations.default, *optionals)
+         if locale.nil?
```

Comment:
Put this logic inside SgtnClient::Config.configurations so that you won't have to repeat it here and elsewhere.

Reply:
As the APIs like getString can be call separately, so seems not good to put the logic in SgtnClient::Config.configurations.